src/Uno.Extensions.Authentication.MSAL/MsalAuthenticationProvider.cs

```diff
@@ -1,7 +1,8 @@
 ﻿#if WINDOWS
-using Microsoft.Identity.Client.Desktop;
+using Microsoft.Identity.Client.Broker;
 #endif
 using Uno.Extensions.Logging;
+using WinRT.Interop;
```

Comment:
This should probably be within the `#if WINDOWS`

Reply:
replaced by https://github.com/unoplatform/uno.extensions/pull/2659#discussion_r1910495906